qiskit/transpiler/passes/scheduling/instruction_alignment.py

```diff
@@ -44,21 +43,39 @@ class AlignMeasures(TransformationPass):
     defines a `pulse granularity`, in other words a data chunk, to allow the DAC to
     perform the signal conversion in parallel to gain the bandwidth.
 
-    Measurement alignment is required if a backend only allows triggering ``measure``
-    instructions at a certain multiple value of this pulse granularity.
-    This value is usually provided by ``backend.configuration().timing_constraints``.
+    A control electronics, i.e. micro-architecture, of the real quantum backend may
+    impose some constraints on the start time of microinstructions.
+    In Qiskit SDK, the duration of :class:`qiskit.circuit.Delay` can take arbitrary
+    value in units of dt, thus circuits involving delays may violate the constraints,
+    which may result in failure in the circuit execution on the backend.
+
+    This pass shifts DAG node start times previously scheduled with one of
+    the scheduling passes, e.g. :class:`ASAPSchedule` or :class:`ALAPSchedule`,
+    so that every instruction start time satisfies alignment constraints described below.
+
+    Pulse alignment constraints
 
-    In Qiskit SDK, the duration of delay can take arbitrary value in units of ``dt``,
-    thus circuits involving delays may violate the above alignment constraint (i.e. misalignment).
-    This pass shifts measurement instructions to a new time position to fix the misalignment,
-    by inserting extra delay right before the measure instructions.
-    The input of this pass should be scheduled :class:`~qiskit.dagcircuit.DAGCircuit`,
-    thus one should select one of the scheduling passes
-    (:class:`~qiskit.transpiler.passes.ALAPSchedule` or
-    :class:`~qiskit.trasnpiler.passes.ASAPSchedule`) before calling this.
+        This value is reported by ``timing_constraints["pulse_alignment"]`` in the backend
+        configuration in units of dt. The start time of the all microwave pulses should be
```

Comment:
in general it may not be "microwave" for other technologies (multiple uses in this paragraph)

Reply:
replaced with pulse instruction